CHARTER.md

```diff
@@ -0,0 +1,108 @@
+  - Code (including types or classes)
+  - UI (including forms)
+  - Databases
+- Relational validation
+- Vocabularies registry
+
+### 1.2: Out-of-Scope
+Neither standards that the JSON Schema project uses (such as JSON and IRI) nor standards or projects that use JSON Schema (such as OpenAPI or AsyncAPI) are in scope, nor does the JSON Schema project have any control over them.
+
+## 2: Relationship with OpenJS Foundation CPC.
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of the JSON Schema project, it is delegated to the JSON Schema Technical Steering Committee ("TSC").
+
+### 2.1 Other Formal Project Relationships
+Section Intentionally Left Blank
+
+## 3: JSON Schema Governing Body (TSC)
+The JSON Schema Technical Steering Committee (TSC) is initially established from the observed major contributors who are currently active and in good standing.
+
+The TSC must have a minimum of four members. There is no maximum TSC membership size.
+
+TSC memberships are not time-limited.
+
+The TSC follows the decision-making process defined in this charter unless otherwise documented.
+
+The TSC aims to work asynchronously. TSC meetings are pre-announced, public, and recorded. Minutes are taken and the recordings made available. If there is a reasonable reason (such as security or privacy), any portion of a meeting, its minutes, and recording, may be kept private.
+
+## 4: Roles & Responsibilities
+
+The JSON Schema project is governed by a Technical Steering Committee (TSC) which is responsible for high-level guidance of the project.
+
+The TSC has final authority over this project including:
+
+- Technical direction
+- Project governance and process (including this policy, though changes to this policy must be approved by the CPC)
+- Contribution policy
+- GitHub repository hosting and administration
+- Establishment of and delegation to working groups or teams
+- Mediating technical conflicts
+
+It is also responsible for establishing a Code of Conduct Committee suitable for mediating non-technical conflicts.
+In any period where such a committee is not yet formed, the TSC must assume temporary responsibility of mediating such conflicts in addition to responsibilities enumerated above.
+
+In joining the TSC, members commit to communicate on a regular basis and respond to issues raised by the TSC in a timely manner. If they are no longer able or willing to make such a commitment, they should discuss this with the TSC or a TSC Chair.
+
+### 4.1 Project Operations & Management
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board or the CPC relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
```

Comment:
Are there any such requirements today that we should be making a note of?

Reply:
This is in reference to the requirement to use a CLA or DCO, and the IP policy: https://ip-policy.openjsf.org

I may be looking to seek an exemption for CLA/DCO, but that remains to be seen.